python/plugins/processing/tests/testdata/qgis_algorithm_tests.yaml

```diff
@@ -191,43 +191,6 @@ tests:
         name: expected/basic_statistics_string.html
         type: file
 
-  # Split lines with lines considers two cases
```

Comment:
I'm curious why these tests need to be removed - as far as I can see the deprecated qgis:splitlineswithlines algorithm hasn't been touched by this PR?